### PR TITLE
Update node-speakers.md

### PR DESCRIPTION
**Pull request by claudineyjr**
Added Claudiney Junior, speaker, writer and video maker about node.js on youtube.

### CODE REVIEW

**Review by ZYSzys — APPROVED** [review]

**Comment by fhemberger** [comment]
@ZYSzys If you have reviewed changes to _speakers/events_ and they are okay, I think it's fine to merge them directly (as they are relatively small changes). No need to wait for me for a second approval. ;)

**Pull request merged**
